### DISCUSSION TITLE
tmrl not installing

### DISCUSSION THREAD

**Discussion by Ahmed2060** [trackmania-rl/tmrl#49] [Q&A]
when i do pip install tmrl 

its say requirement fulfilled but i cant find TmrlData file

**Comment by yannbouteiller** [comment]
This is strange, what OS are you using?

On windows it should be under `C:\Users\your_username`

Or you can execute this python script to find where it is on your PC:

```python
from pathlib import Path
HOME_FOLDER = Path.home()
TMRL_FOLDER = HOME_FOLDER / "TmrlData"
print(TMRL_FOLDER)
```

**Reply by Ahmed2060** [reply]
I am using windows, I tried the script and it shows me that it is where its supposed to be 

![Screenshot 2023-05-14 101251](https://github.com/trackmania-rl/tmrl/assets/38594424/d85ceddc-0b2f-442f-af15-5eb89728ed0f)

but when I go there I cant find it as if its hidden or smt 

![Screenshot 2023-05-14 101205](https://github.com/trackmania-rl/tmrl/assets/38594424/b78f2ee2-5fa5-4a16-b95c-2cdc9b1cf01c)

**Reply by Ahmed2060** [reply]
![image](https://github.com/trackmania-rl/tmrl/assets/38594424/362d4701-8b53-4747-a881-26dd5e497e7f)

**Comment by Ahmed2060** [comment]
I just told a friend to send me the file

**Comment by yannbouteiller** [comment]
This is strange, maybe pip did not have the permission to create this folder on your machine?

**Comment by yannbouteiller** [comment]
The bit of code responsible for creating this folder is [here](https://github.com/trackmania-rl/tmrl/blob/b142b8ec95ef7ca59a89a23e6ec22795e842733d/setup.py#L39) btw.

Maybe a warning was raised by pathlib and suppressed by pip?

For people who may have this issue in the future, your can find the `resources.zip` file in release assets. For instance, the current `resources.zip` is in [release 0.4.2](https://github.com/trackmania-rl/tmrl/releases/tag/v0.4.2).

**Comment by NDR008** [comment]
I think I had this issue and manually created that folder from the resource
file.
